test/features/events/events.js

```diff
@@ -21,7 +21,7 @@ describe('Change event emitting', function() {
         });
     });
 
-    function createEventLogging(done, topic, uri) {
+    function createEventLogging(done, topic, uri, trigger) {
```

Comment:
Do you think it would make sense to introduce an "options" or "eventdata" object instead, to avoid the parameter list from growing too much when we add more optional bits of information in the future?